addons/GameTemplate/Localization/Localization.csv

```diff
@@ -57,17 +57,19 @@ SUBMIT,Submit,Enviar
 SIGN UP,Sign Up,Registrarse
 LOADING_SCORES,Loading scores…,Cargando puntuaciones…
 NO_SCORES_YET,No scores yet!,Sin puntuaciones por el momento
-INTRO-1,"Back in 2022 year, B.O.B. the robot was sended\nto the Moon with the purpose of terraforming it,\nbut contact was lost.","En el año 2022 el robot B.O.B. fue desplegado\nen la Luna con propósito de terraformarla.\nPero se perdió todo contacto con él."
-INTRO-2,"The information that B.O.B. has gathered is\ncrutial to know humanity's chances of being established\nin the Moon, due to the inminent end of our planet, the Earth.\nThat's way the robot L.A.R.R.Y. is being sended\nto save B.O.B. or, at least, it's information...","La información que B.O.B. pudo conseguir\nes crucial para saber las chances de la humanidad de establecerse\nen la Luna debido al poco tiempo que le queda a la Tierra.\nEs por ello que L.A.R.R.Y. fue desplegado para encontrar\na B.O.B. o por lo menos lo que sabía…"
-INTRO-3,"L.A.R.R.Y. unit, launched. Target: localize\nand recover B.O.B.'s parts as much as possible.\nLimitation: There's not enough fuel. In five minutes, our\nspaceship will come back for L.A.R.R.Y. Good luck.","Unidad L.A.R.R.Y. desplegada. Objetivo: localizar y recuperar\nla mayor cantidad de piezas de B.O.B. posibles.\nLimitación: No hay suficiente combustible. En 5 minutos la nave\nrecuperará a L.A.R.R.Y con todo lo que haya obtenido.\nBuena suerte."
+INTRO-1,"Back in 2022 year, B.O.B. the robot was sent to the Moon with the purpose of terraforming it, but contact was lost.","En el año 2022 el robot B.O.B. fue desplegado en la Luna con propósito de terraformarla. Pero se perdió todo contacto con él."
+INTRO-2,"The information that B.O.B. had gathered is crutial to know humanity's chances of being established in the Moon, due to the inminent end of our planet, the Earth.","La información que B.O.B. pudo conseguir es crucial para saber las chances de la humanidad de establecerse en la Luna debido al poco tiempo que le queda a la Tierra."
+INTRO-3,"That's why the robot L.A.R.R.Y. is being sent to save B.O.B. or, at least, it's information...","Es por ello que L.A.R.R.Y. fue desplegado para encontrar a B.O.B. o por lo menos lo que sabía…"
+INTRO-4,"L.A.R.R.Y. unit, launched. Target: localizeand recover B.O.B.'s parts as much as possible. Limitation: There's not enough fuel.","Unidad L.A.R.R.Y. desplegada. Objetivo: localizar y recuperarla mayor cantidad de piezas de B.O.B. posibles.Limitación: No hay suficiente combustible."
```

Comment:
localizeand typo

Reply:
no entendi este comentario, sé más específico por favor.

Reply:
pusiste "Target: ... localizedand ...". 
Un typo man.

Reply:
ahh bolu pense qe era localize and typo jajajaja